go/pserver/client_test.go

```diff
@@ -31,7 +30,7 @@ func init() {
 		port[i] = p
 
 		go func(l net.Listener) {
-			s, err := pserver.NewService("", time.Second*5)
+			s, err := pserver.NewService(0)
```

Comment:
Maybe if develop is confused, he can check the definition of `pserver.NewService` :) Please let me know if you think otherwise.